bot.rb

```diff
@@ -1,8 +1,9 @@
-require 'octokit'
-require 'pry'
-require 'excon'
 require 'colored'
+require 'date'
+require 'excon'
 require 'json'
+require 'octokit'
+require 'pry'
```

Comment:
Is this needed?

Reply:
I just sorted the `require`s cause rubocop complained. The only line I added is `require 'date'`

Reply:
Sorry, thought you added it.